README.md

```diff
@@ -1,11 +1,31 @@
 # Coursework
 
-Like learning a musical instrument, programming requires daily practise.
+Like learning a musical instrument, programming requires daily practice.
 
-The exercises are split into three folders: `exercises`, `mandatory` and `extra`. All homework in the `exercise` and `mandatory` section **must** be completed for homework by the following lesson. 
+The exercises are split into two folders: `mandatory` and `extra`. All homework in the `mandatory` section **must** be completed for homework by the following lesson.
 
 The `extra` folder contains exercises that you can complete to challenge yourself, but are not required for the following lesson.
 
+If you think you need to do more practice with the basics, then you can find some more exercises listed in `resources.md`. These exercises are not mandatory and won't be assigned as part of your coursework for the week.
```

Comment:
Deliberately not listed them ( exercises ) here so trainees don't view them as mandatory.